react-frontend-app/src/components/BarGraph.js

```diff
@@ -0,0 +1,217 @@
+// import all packages
+import reportWebVitals from "./../reportWebVitals.js";
+import React, { Component } from "react";
+import { max, descending, schemeCategory10, version, schemePastel1 } from "d3";
+import { scaleOrdinal, scaleLinear } from "d3-scale";
+import { axisLeft } from "d3-axis";
+import { select, selectAll } from "d3-selection";
+import { format } from "d3-format";
+import '../style/App.css';
+
+// import axios from "axios";
+// goal is to pass in data from the api call (currently, this is dummy data)
+var data = {
+    "white": {
+        "bisg": "0.6700283648",
+        "zest": "0.6180442684"
+    },
+    "black": {
+        "bisg": "0.0033692506",
+        "zest": "0.0153741615"
+    },
+    "api": {
+        "bisg": "0.309192341",
+        "zest": "0.3543504815"
+    },
+    "native": {
+        "bisg": "0.0019506335",
+        "zest": "0.0010750276"
+    },
+    "multiple": {
+        "bisg": "0.0064047544",
+        "zest": "0.0093698772"
+    },
+    "hispanic": {
+        "bisg": "0.0090546558",
+        "zest": "0.0017861839"
+    }
+};
+
+const d3 = {
+    select,
+    selectAll,
+    descending,
+    schemeCategory10,
+    version,
+    scaleOrdinal,
+    scaleLinear,
+    max,
+    axisLeft,
+    format,
+    schemePastel1
+};
+
+class BarGraph extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.createBar = this.createBar.bind(this);
+    }
+
+    async componentDidMount() {
+        // const res = await axios.get('http://0.0.0.0:5000/surgeo');
+        // const data = await res.json();
+        
+        this.createBar(data);
+    }
+
+    componentDidMount() {
```

Comment:
Just curious. Is there a reason that componentDidMount is listed twice?

Reply:
That was an accident, oops!